templates/default.json

```diff
@@ -62,6 +62,7 @@
       "collectCoverageFrom": ["**/*.{js,jsx,ts,tsx}"],
       "coveragePathIgnorePatterns": [
         "<rootDir>/build/",
+        "<rootDir>/src/pages/_document.jsx",
```

Comment:
Shouldn't be ignored - not a good example to set. If you look at the client-internal repo forked from this, I believe I have an example of testing the _app.jsx or _document.jsx